chart/templates/NOTES.txt

```diff
@@ -33,6 +33,14 @@ DEPRECATION WARNING:
 
 {{- end }}
 
+{{- if .Values.ingress.web.hosts }}
```

Comment:
This should only apply if the `airflowVersion` is 3+ - the chart still fully supports Airflow 2.